packages/components/buttons/icon-button/src/icon-button.styles.ts

```diff
@@ -3,12 +3,6 @@ import { designTokens } from '@commercetools-uikit/design-system';
 import { css } from '@emotion/react';
 import type { TIconButtonProps } from './icon-button';
 
-const buttonSizes = {
-  small: '16px',
-  medium: '24px',
-  big: '32px',
-};
```

Comment:
We moved this values to the design-systems definitions so we can have different ones per theme.